intermittent_failures.md

```diff
@@ -0,0 +1,138 @@
+
+1. Determine if the failure is intermittent
+   - If possible, execute on your local machine multiple times to tally a failure rate, and attempt to debug the issue
+   - Execute the Jenkins pipeline as many as 10 times, and save at least one text file with a failing log
+   - If the failure occurs consistently ***DO NOT CONTINUE*** following these steps, and instead reach out over [Discord](https://discord.gg/p3padwr58u)
+2. [Cut a GitHub Issue](https://github.com/o3de/o3de/issues/new/choose) to track both fixing the underlying issue and re-enabling the test
+   - Add a label for the SIG which maintains that feature 'sig/XYZ' ***OR*** use label 'needs-sig'
+3. The source of intermittent failure should be immediately removed from the pipeline
+   - If possible, submit a new pull request which reverts or disables the intermittent module, feature, test, or hardware-configuration
+
+### Exceptions
+
+The policy above applies to all code, scripts, and hardware used in the Automated Review Pipeline with the following exceptions:
+
+- It is rarely more risky to remove an unstable critical test (or feature) than to leave it in a known intermittently failing state
+  - If you have a reason to not follow this policy, please reach out to SIG-Testing on [Discord](https://discord.gg/p3padwr58u)!
+- Intermittent failures ocurring on only one Operating System are still intermittent failures, though it is reasonable to conditionally disable
+- Builds or tests which reliably fail at a given commit are **not** considered intermittent
+  - Such failures may appear inconsistent across multiple commits, when thrashed by rapid code changes
+  - While the pipeline makes these difficult to introduce, when they occur it is still appropriate to immediately revert or disable until fixed
+
+## Disabling Individual Tests
+
+To quickly disable a specific test, modify its source as follows:
+
+### GoogleTest
+
+Change tests of the form:
+
+```cpp
+TEST(MyTestClassName, MyTestUnitOfWork_NotableContext_ExpectedResult)
+
+TEST_F(MyTestFixutreName, MyTestUnitOfWork_NotableContext_ExpectedResult)
+
+TEST_P(MyTestParameterizedFixutreName, MyTestUnitOfWork_NotableContext_ExpectedResult)
+```
+
+...to add `DISABLED` to their name:
+
+```cpp
+TEST(MyTestClassName, DISABLED_MyTestUnitOfWork_NotableContext_ExpectedResult)
+
+TEST_F(MyTestFixutreName, DISABLED_MyTestUnitOfWork_NotableContext_ExpectedResult)
+
+TEST_P(MyTestParameterizedFixutreName, DISABLED_MyTestUnitOfWork_NotableContext_ExpectedResult)
+```
+
+### PyTest
+
+Change any test:
+
+```python
+def test_MyTestUnitOfWork_NotableContext_ExpectedResult():
+```
+
+...to add a skip marker to the test
+
+```python
+@pytest.mark.skip(reason="reason this is disabled, such as an issue tracked in GitHub")
+def test_MyTestUnitOfWork_NotableContext_ExpectedResult():
+```
+
+## The Sandbox Test Suite
```

Comment:
I have a slight preference to move this after the Troubleshooting section as I think it's more important to be seen